pde/pom.xml

```diff
@@ -47,7 +47,7 @@
     <repository>
       <id>m2e</id>
       <layout>p2</layout>
-      <url>https://download.eclipse.org/technology/m2e/releases/latest/</url>
+      <url>https://download.eclipse.org/technology/m2e/releases/1.20.1/</url>
```

Comment:
Roll back this line to use the latest m2e?

Reply:
done